### ISSUE TITLE
Does TPOT support multi-label classification?

### ISSUE THREAD

**Issue by 100330706** [EpistasisLab/tpot#856]
All the posts I have seen about multi-label classification are pretty old so I was wondering if TPOT currently supports it.

Regards.

**Comment by weixuanfu** [comment]
No, TPOT does not support multi-label classification so far.

**Comment by jhmenke** [comment]
~isn't the MNIST example multi-label classification already?~
I see now that it's strictly predicting one label only

**Comment by weixuanfu** [comment]
@jhmenke The [MNIST (from scikit-learn)](https://scikit-learn.org/stable/datasets/index.html#digits-dataset) is not a multi-label classification but a multi-class classification.

**Comment by eddiebergman** [comment]
Any update on whether this is a planned feature?

As far as I'm aware multi-label is only supported by a subset of sklearn algorithms and some require setting specific options in their `__init__` to enable it which seems to go against how TPOT is set up to specify pipelines. 

However, some sklearn algorithms automatically infer this and adapt accordingly without any extra specification, perhaps TPOT could do something similar in it's `_init_pretest`, detecting multi-label labels and reducing the configuration space to just those algorithms which require no extra steps to support it?

https://scikit-learn.org/stable/modules/multiclass.html

**Comment by eddiebergman** [comment]
Apologies, I see this feature was started and paused in PR #1001. I'm unlikely to find time for this in the near future but leaving this here to let future visitors of the thread know the current status.
